extensions/core/interfaces/code/code.css

```diff
@@ -0,0 +1,14 @@
+.CodeMirror {
+  /*color: var(--gray);*/
+  font-weight: inherit;
+  /*height: auto;*/
```

Comment:
Other nitpicky thing: I'd like to remove code instead of comment it out. We have git to rely on if we ever want to revert back to code. Removing deprecated code keeps it nice and clean.